lib/absinthe/phase/document/validation/arguments_of_correct_type.ex

```diff
@@ -79,7 +79,14 @@ defmodule Absinthe.Phase.Document.Validation.ArgumentsOfCorrectType do
     node.fields
     |> Enum.flat_map(fn
       %{flags: %{invalid: _}, schema_node: nil} = child ->
-        [unknown_field_error_message(child.name)]
+        field_suggestions =
+          case node.schema_node do
```

Comment:
You should `Type.unwrap(node.schema_node)` because this could be wrapped in `non_null`.

Reply:
Used this line as example, should I change this one also? It's not related with the new feature!